scripts/install_helpmakego.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+HELPMAKEGO_VERSION="$1"
+HELPMAKEGO=bin/${HELPMAKEGO_VERSION}/helpmakego
+
+if ! [ -x ${HELPMAKEGO} ]; then
+    echo "Installing helpmakego@${HELPMAKEGO_VERSION}"
+    go install -mod readonly "github.com/iwahbe/helpmakego@${HELPMAKEGO_VERSION}"
```

Comment:
I wonder if it works if we set `GOPATH=bin/${HELPMAKEGO_VERSION}/` here to avoid updating the global `$GOPATH/bin/helpmakego` ?

Reply:
Yes found a way to not clobber $GOPATH/bin/helpmakego.